### PR TITLE
Snowflake handle empty sql list

### PR DESCRIPTION
**Pull request by pankajastro**
closes: https://github.com/astronomer/astronomer-providers/issues/433

### CODE REVIEW

**Comment by park-peter** [comment]
this will conflict with #810 as it changes the way SnowflakeHook returns the result to be in line with the non-deferrable hook. Please wait until #810 is merged.

**Review by rajaths010494 — APPROVED** [review]

**Pull request merged**
